ui/src/views/404/Footer/index.jsx

```diff
@@ -0,0 +1,58 @@
+// @flow
+
+// This is an example of styles that are defined inline
+const style = {
+    borderTopWidth: '3px',
+    borderColor: '#0061A6',
+    borderTopStyle: 'solid',
+    paddingTop: '2px',
+    fontSize: '11px',
+    lineHeight: '14px',
+    margin: '15px 0',
+    clear: 'both',
+};
+
+const item = ({ text, ...props }) =>
+    <li
+        style={{
+            display: 'inline',
+            ':after': {
+                content: '" | "',
+            },
+        }}>
+        <a
+            {...props}
+            style={{
+                color: '#005689',
+                textDecoration: 'none',
+            }}>
```

Comment:
I agree with the point made by Natalia and would add the concern, that we don't think about naming structure anymore, but just add new stuff as it comes (with lots of duplication which is hard to read and to maintain).